src/gridcoin/upgrade.cpp

```diff
@@ -152,6 +160,11 @@ bool Upgrade::CheckForLatestUpdate(bool ui_dialog, std::string client_message_ou
     client_message_out.append(_("Github version: ") + GithubReleaseData + "\r\n");
     client_message_out.append(_("This update is ") + GithubReleaseType + "\r\n\r\n");
 
+    if (NewMandatory)
+    {
+        client_message_out.append(_("Your wallet version is LOWER than the latest mandatory. You MUST UPGRADE YOUR WALLET!\n"));
```

Comment:
In UI design, uppercase text content is considered a faux pas. I understand that we want to communicate a sense of urgency. Perhaps we can reword this into a message that some may perceive as more respectful:

```suggestion
        client_message_out.append(_("WARNING: A mandatory release is available. Please upgrade as soon as possible.\n"));
```

Reply:
haha.. I guess yelling isn't such a good idea.... :)